src/client.js

```diff
@@ -0,0 +1,52 @@
+import isString from 'lodash/isString';
+import { Client as CoreClient, KinveyError, isDefined } from 'kinvey-js-sdk/dist/export';
+let storage = Ti.App.Properties;
+
+class ActiveUserStorage {
+  get(key) {
+    if (!isString(key)) {
+      throw new KinveyError('ActiveUserStorage key must be a string.');
+    }
+
+    try {
+      return JSON.parse(storage.getString(key));
+    } catch (e) {
+      return null;
```

Comment:
@thomasconner same as my comment on html5 repo - this should, in the very least, log an unexpected error.